installation/cmd/run.ps1

```diff
@@ -5,6 +5,8 @@ param (
     [string]$PASSWORD = ""
 )
 
+Write-Output @"The run.ps1 script is deprecated and will be removed with Kyma release 1.4. Use Kyma CLI instead"@

```

Comment:
Similarly to the docs, I'd re-design the deprecation message to not include the cutoff date. 
I suggest something along the lines of: 
`This script is deprecated. Use the Kyma CLI instead.`